data/abi/abi_type.go

```diff
@@ -113,15 +113,15 @@ func (t Type) String() string {
 var staticArrayRegexp = regexp.MustCompile(`^([a-z\d\[\](),]+)\[([1-9][\d]*)]$`)
 var ufixedRegexp = regexp.MustCompile(`^ufixed([1-9][\d]*)x([1-9][\d]*)$`)
 
-// TypeFromString de-serialize ABI type from a string following ABI encoding.
-func TypeFromString(str string) (Type, error) {
+// TypeOf de-serialize ABI type from a string following ABI encoding.
```

Comment:
nit: suggested docstring improvement/simplification
```suggestion
// TypeOf parses an ABI type string.
// For example: `TypeOf("(uint64,byte[])")`
```